src/styles/button.scss

```
@@ -0,0 +1,8 @@
+.button {
+  background: #cd4d31;
+  padding: 20px 0;
+  width: 270px;
+  border: 1px solid #cd4d31;
+
+  color: white;
```

```suggestion
  padding: 20px 0;
  width: 270px;
  
  background: #cd4d31;
  border: 1px solid #cd4d31;
  color: white;
```

ти використовуєш #cd4d31; мінімум два рази цей колір тільки тут, доречно в змінну